lib_comfyui/ipc_strategies.py

```diff
@@ -0,0 +1,110 @@
+import contextlib
+import dataclasses
+import os
+import pickle
+from multiprocessing.shared_memory import SharedMemory
+from typing import IO, Union
+
+
+__all__ = [
+    'FileSystemIpcStrategy',
+    'SharedMemoryIpcStrategy',
+    'OsFriendlyIpcStrategy',
+]
+
+
+class FileSystemIpcStrategy:
```

Comment:
do you think this and `SharedMemoryIpcStrategy` would benefit from having a superclass?

Reply:
yeah